packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -91,12 +91,7 @@ enum HostArtifact {
   webPlatformDart2JSSoundKernelDill,
 
   /// The precompiled SDKs and sourcemaps for web debug builds.
-  webPrecompiledSdk,
```

Comment:
Is this cleanup related to the flag removal?

Reply:
Yeah, these are the unsound parts of the web sdk, which are no longer reachable.

Reply:
This https://github.com/flutter/flutter/issues/122215 is tracking not building it